charts/stable/hastebin/chart/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+appVersion: 1.0.0
+description: haste helm package
+name: haste
+version: 1.1.0
+kubeVersion: ">=1.16.0-0"
+keywords:
+- haste
+- hastebin
+- haste-server
+- pastebin
+home: https://github.com/k8s-at-home/charts/tree/master/charts/stable/haste
+icon: https://github.com/seejohnrun/haste-server/blob/master/static/logo.png
+sources:
+- https://github.com/seejohnrun/haste-server
+maintainers:
+- name: rwaltr
+  email: rwalt@pm.me
+dependencies:
+- name: common
+  repository: https://library-charts.k8s-at-home.com
+  version: 2.2.0
```

Comment:
Update this to latest common library version

Reply:
updated in https://github.com/k8s-at-home/charts/pull/800/commits/769f44f54a9f96ea280834e666a3c0e2d46314b0